Swift/BRCrypto/BRCryptoWallet.swift

```diff
@@ -172,6 +187,12 @@ public final class Wallet: Equatable {
         }
     }
 
+    public func estimateFeeToImportFrom (key: Key,
+                                         fee: NetworkFee,
+                                         completion: @escaping (Result<TransferFeeBasis,FeeEstimationError>) -> Void) {
+        precondition (false)
```

Comment:
We have JIRA issues for implementing these?

Reply:
Not at the moment. I can create now. Had been waiting for this to get in.